projects/core/src/user/services/order-page-meta.resolver.ts

```diff
@@ -0,0 +1,121 @@
+import { Injectable } from '@angular/core';
+import { combineLatest, defer, Observable, of } from 'rxjs';
+import {
+  distinctUntilChanged,
+  map,
+  shareReplay,
+  switchMap,
+} from 'rxjs/operators';
+import { BreadcrumbMeta, PageRobotsMeta } from '../../cms/model/page.model';
+import { BasePageMetaResolver } from '../../cms/page/base-page-meta.resolver';
+import { PageMetaResolver } from '../../cms/page/page-meta.resolver';
+import {
+  PageBreadcrumbResolver,
+  PageRobotsResolver,
+  PageTitleResolver,
+} from '../../cms/page/page.resolvers';
+import { TranslationService } from '../../i18n/translation.service';
+import { PageType } from '../../model/cms.model';
+import { RoutingService } from '../../routing/facade/routing.service';
+
+/**
+ * Resolves the page data for Order Pages.
+ *
+ * Breadcrumbs are built in this implementation only.
+ *
+ * @property {string} ORDER_SEMANTIC_ROUTE the default root path for Order pages.
+ * @property {string} ORDER_TRANSLATION_KEY the default i18n key for the Order breadcrumb label.
+ * @property {string} ORDER_HOME_PATH the default path for Order home poge.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class OrderPageMetaResolver
+  extends PageMetaResolver
+  implements PageBreadcrumbResolver, PageTitleResolver, PageRobotsResolver {
+  /**
+   * Translation key for the breadcrumb of Order home page
+   */
+  protected readonly ORDER_TRANSLATION_KEY = 'orderDetails.breadcrumb';
+
+  /**
+   * The semantic route of the Order Details page. It's used to recognize whether
+   * we are on this page. In such a case we are adding additional breadcrumb element.
+   */
+  protected readonly ORDER_DETAILS_SEMANTIC_ROUTE = 'orderDetails';
+
+  /**
+   * The semantic route of the Orders landing page.
+   */
+  protected readonly ORDER_SEMANTIC_ROUTE = 'orders';
+
+  constructor(
+    protected basePageMetaResolver: BasePageMetaResolver,
+    protected routingService: RoutingService,
+    protected translation: TranslationService
+  ) {
+    super();
+    this.pageType = PageType.CONTENT_PAGE;
+    this.pageTemplate = 'AccountPageTemplate';
```

Comment:
Same issue as with SavedCartPageMetaResolver. Please wait for https://github.com/SAP/spartacus/pull/12569 to  be implemented